packages/sites/src/drafts/types.ts

```diff
@@ -0,0 +1,3 @@
+import { IModel } from "@esri/hub-common";
+
```

Comment:
we have a types package... I think use that?

Reply:
Types package is pretty sparse. Do you want me to move the types currently in `hub-common` over as part of this PR? @dbouwman